app/models/user.rb

```diff
@@ -498,26 +498,30 @@ def number_articles_per_day
     article_count = user.page_views.select(:article_id).distinct.count
     return article_count.round(1) if user.created_at.today?
 
-    (article_count.to_f / (Time.zone.today - user.created_at.to_date).to_i).round(1)
+    (article_count.to_f / (Time.zone.today - user.created_at.to_date).to_i).round
```

Comment:
additional decimal places don't bring meaningful information for this stat, rounding to the closest integer